Assets/Simon/Scripts/Colors.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum Colors
+{
+    Green,

```

Comment:
you can put this in another file that's more relevant, such as ColorButtonEffectPlayer

Reply:
OK done. Is this a best practice to move enums to a class file? If you look for it in the project folders, isn't it easier to find if it is in its own file?